tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt

```diff
@@ -17,50 +27,123 @@ list(REMOVE_ITEM MICRO_SOURCES ${MICRO_TESTS_SOURCES})
 list(REMOVE_ITEM MICRO_KERNELS_SOURCES ${MICRO_TESTS_KERNELS_SOURCES})
 list(REMOVE_ITEM MICRO_MEMORY_SOURCES ${MICRO_TESTS_MEMORY_SOURCES})
 
-add_subdirectory(src)
-add_subdirectory(cmake_sub/simple_features)
-add_subdirectory(cmake_sub/micro_features)
-add_subdirectory(cmake_sub/micro_speech)
-add_subdirectory(cmake_sub/kiss_fft)
-add_subdirectory(cmake_sub/microfrontend)
+target_compile_options(app PRIVATE "-fno-threadsafe-statics")
 
-target_sources (
-	app PRIVATE
-        ${MICRO_SOURCES}
-        ${MICRO_KERNELS_SOURCES}
-        ${MICRO_MEMORY_SOURCES}
-        ${MICRO_HEADERS}
-        ${MICRO_KERNEL_HEADERS}
-        ${MICRO_MEMORY_HEADERS}
-        ${TF_MICRO_DIR}/../c/common.c
-        ${TF_MICRO_DIR}/../core/api/error_reporter.cc
-        ${TF_MICRO_DIR}/../core/api/flatbuffer_conversions.cc
-        ${TF_MICRO_DIR}/../core/api/op_resolver.cc
-        ${TF_MICRO_DIR}/../core/api/tensor_utils.cc
-      	${TF_MICRO_DIR}/../kernels/internal/quantization_util.cc
-        ${TF_MICRO_DIR}/../kernels/kernel_util.cc
-	)
-
-target_link_libraries(
-	app PRIVATE
-	zephyr_src
-	micro_features
-	simple_features
-	micro_speech
-	kiss_fft
-	microfrontend
-	)
+target_sources(
+  app
+  PRIVATE ${MICRO_SOURCES}
+          ${MICRO_KERNELS_SOURCES}
+          ${MICRO_MEMORY_SOURCES}
+          ${MICRO_HEADERS}
+          ${MICRO_KERNEL_HEADERS}
+          ${MICRO_MEMORY_HEADERS}
+          ${TF_LITE_DIR}/c/common.c
+          ${TF_LITE_DIR}/core/api/error_reporter.cc
+          ${TF_LITE_DIR}/core/api/flatbuffer_conversions.cc
+          ${TF_LITE_DIR}/core/api/op_resolver.cc
+          ${TF_LITE_DIR}/core/api/tensor_utils.cc
+          ${TF_LITE_DIR}/kernels/internal/quantization_util.cc
+		  ${TF_LITE_DIR}/kernels/kernel_util.cc
+		  
+          # zephyr-src
+          src/audio_provider.cc
+          src/audio_provider.h
+          src/command_responder.cc
+		  src/command_responder.h
+		  
+		  # micro_speech 
+		  # ${MICRO_SPEECH}/command_responder.cc
+          # ${MICRO_SPEECH}/command_responder.h
+		  # ${MICRO_SPEECH}/audio_provider.cc
+		  # ${MICRO_SPEECH}/audio_provider.h
+          ${MICRO_SPEECH}/feature_provider.cc
+          ${MICRO_SPEECH}/feature_provider.h
+          ${MICRO_SPEECH}/main.cc
+          ${MICRO_SPEECH}/main_functions.cc
+          ${MICRO_SPEECH}/main_functions.h
+          ${MICRO_SPEECH}/recognize_commands.cc
+          ${MICRO_SPEECH}/recognize_commands.h
+          ${MICRO_SPEECH}/no_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/no_1000ms_sample_data.h
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.cc
+          ${MICRO_SPEECH}/yes_1000ms_sample_data.h
+          # ${MICRO_SPEECH}/no_30ms_sample_data.cc
+          # ${MICRO_SPEECH}/no_30ms_sample_data.h
```

Comment:
What are those comments? Do we need this?

Reply:
Yup. We need them, it is still part of the micro_speech source or ready for use audio samples (for testing purposes). In order to have smaller binary I've commented them for now.

Reply:
Please resolve this if everything is clear